packages/story-editor/src/components/panels/design/pageAttachment/pageAttachment.js

```diff
@@ -285,6 +307,39 @@ function PageAttachmentPanel() {
               {__('Link icon', 'web-stories')}
             </Text>
           </StyledRow>
+          <Row>
```

Comment:
If it's exactly the same here as in the Link panel, should we perhaps reuse instead of duplicating? Maybe even combine it with the `LinkInput` component?

Reply:
The link and page attachment panels are like this in general. It should be more generic component. 

Anyway, I have refactored the logic to be it's own component.